clap_blocks/src/querier.rs

```diff
@@ -36,6 +37,15 @@ pub enum Error {
         shard_index: ShardIndex,
         name: Arc<str>,
     },
+
+    #[snafu(display("Could not parse ingester URI `{uri}`: {source}"))]
+    CouldNotParseIngesterUri {
+        uri: String,
+        source: <Uri as FromStr>::Err,
```

Comment:
I think this could name the type explicitly as well, if that were easier on the eyes:

```suggestion
        source: http::uri::InvalidUri,
```

https://docs.rs/http/0.2.8/http/uri/struct.Uri.html#impl-FromStr